stats/game.py

```diff
@@ -1,5 +1,9 @@
 from models import Game
 from constants import team_abbrev_mapping
+from collections import namedtuple
+
+
+GameEntry = namedtuple("GameEntry", "season_id, game_id, game_date, matchup_in, winner, loser")
```

Comment:
Pythonic touch. It's cleaner and self documenting.

Data classes is another approach, but I was not sure which version of Python you were currently targeting or using.

Reply:
I guess I am going for 3.8. Are data classes available in 3.8? This does seem ike a better defined datastructure though.

Reply:
3.8 is good, I think they were introduced in 3.7